src/idiomatic/foundations-api-design/meaningful-doc-comments/avoid-redundancy.md

```diff
@@ -0,0 +1,102 @@
+---
+minutes: 15
+---
+
+# Avoiding Redundancy
+
+Names and type signatures communicate a lot of information, don't repeat it in
+comments!
+
+```rust,compile_fail
```

Comment:
I'm wondering if we can do something about the many `compile_fail` attributes introduced here. I count 29 new ones in this PR: that's 29 code blocks where we have no assurance that the code we show the students is correct 😅

Instead of using `IpAddrV4` below, perhaps do

```rust
fn business_asset_as_string(asset: &BusinessAsset) -> String { ... }
```

and thus lean into the other types you define. Or make an example that uses more primitive types.